### DISCUSSION TITLE
How to interpolate from 6-tile FV3 cubed sphere grids to global Gaussian grid

### DISCUSSION THREAD

**Discussion by mdtoyNOAA** [dtcenter/METplus#1435] [Configuration] [answered]
Is there a METplus tool that interpolates fields from the 6-tile FV3 cubed sphere grids to the global Gaussian grid? Specifically, what I would like to do is interpolate the "stddev" field from the six "INPUT/oro_data.tilex.nc" cubed sphere grid files to the Gaussian grid. Thank you.

**Comment by TaraJensen** [comment, marked as answer]
Michael,

I see you are wanting to work with the FV3 output but on a Gaussian grid.  METplus does not have the capacity to work with the native output at this time.  We rely on users running the files through the Unified Post Processor (UPP) to do the post-processing and interpolation to a standard grid.  At this time, DTC supports UPP to the community and it should be readily available on the NOAA platforms you work on.  Here's a link to the DTC UPP website: [https://dtcenter.org/community-code/unified-post-processor-upp](https://dtcenter.org/community-code/unified-post-processor-upp).  If you need further assistance, please feel free to reach out to the DTC staff at GSL to help.

Hope this helps.

**Comment by mdtoyNOAA** [comment]
Hi Tara,

Thank you for getting back to me and for the UPP lead.

Best regards,
Mike
